kolibri/utils/system.py

```diff
@@ -58,9 +59,11 @@ def _kill_pid(pid, softkill_signal_number):
             "Soft kill signal could not be sent (OSError); process may not exist?"
         )
         return
-    # give some time for the process to clean itself up gracefully bfore we force anything
+    if pid_exists(pid):
+        logger.info("Waiting for Kolibri to finish shutting down")
```

Comment:
this logging will be useful. Recommend updating logging below `if pid_exists(pid):` to either `info` or `warning`